extensions/0znzw/0tils.js

```diff
@@ -0,0 +1,1565 @@
+          opcode: 'Setup',
+          blockType: Scratch.BlockType.COMMAND,
+          text: 'Setup Librarys | [log]',
+          arguments: {
+            log: {
+              type: Scratch.ArgumentType.BOOLEAN,
+              defaultValue: false
+            }
+          }
+        }],
+        menus: {
+          EVENT_FIELD: {
+            items: ['onetime', 'every second']
+          },
+          EVENT_FIELD_MIN: {
+            items: ['onetime']
+          },
+          cryptmodes: {
+            items: ['Encrypt', 'Decrypt']
+          },
+          aesbytes: {
+            items: ['128', '256']
+          },
+          geomodes: {
+            items: ['Latitude', 'Longitude']
+          },
+          i36modes: {
+            items: ['text', 'number']
+          },
+          encodings: {
+            items: [{
+              text: 'base64(browser)',
+              value: 'base64_old'
+            }, 'unicode', 'binary', 'cloud']
+          },
+          hashes: {
+            items: ['md5', 'sha1',
+            //'sha128', 
+            'sha256', 'sha512', {
+                text: '32-Bit Fnv-1a',
+                value: 'Fnv32'
+              },
+              /*{
+                text: 'Jenkins One at a Time',
+                value: 'JOaaT'
+              },*/
+              'TinySimpleHash', 'cyrb53', '147030016'
+            ]
+          },
+          recodes: {
+            items: ['Encode', 'Decode']
+          },
+          bools: {
+            acceptReporters: true,
+            items: [{
+              text: 'true',
+              value: 'true'
+            }, {
+              text: 'false',
+              value: 'false'
+            }]
+          },
+          opentargets: {
+            items: [{
+              text: 'newtab',
+              value: '_blank'
+            }, {
+              text: 'this tab (redirect)',
+              value: '_self'
+            }, '_parent', '_top']
+          },
+          mathnos: {
+            acceptReporters: true,
+            items: [{
+              text: 'newline',
+              value: '\n'
+            }, {
+              text: 'φ',
+              value: 1.6180339887
+            }, {
+              text: 'e',
+              value: 2.7182818284
+            }, {
+              text: '∞',
+              value: Infinity
+            }, {
+              text: 'π',
+              value: 3.1415926535
+            }, {
+              text: 'NaN',
+              value: 'NaN'
+            }, {
+              text: 'undefined',
+              value: 'undefined'
+            }, {
+              text: 'null',
+              value: 'null'
+            }]
+          }
+        }
+      };
+    }
+    getVersion() {
+      return _SGCUtils.version;
+    }
+    //SETUP OF EXTERNAL LIBRARYS!! (hidden but still usable)
+    /* eslint-disable */
+    Setup(args) {
+      // YES WE KNOW, IT LEAKED A SCRIPT TO GLOBAL SCOPE, BUT WE COULD NOT GET IT TO WORK OUTSIDE ITS OWN SCRIPT TAG, SORRY.
```

Comment:
you will have to figure out how to make that work

Reply:
could i just remove it until then?

Reply:
i hid the blocks and made it just return nothing